storage/database/db_manager.go

```diff
@@ -1072,13 +1072,12 @@ func (dbm *databaseManager) PutBodyToBatch(batch Batch, hash common.Hash, number
 
 // WriteBodyRLP stores an RLP encoded block body into the database.
 func (dbm *databaseManager) WriteBodyRLP(hash common.Hash, number uint64, rlp rlp.RawValue) {
+	dbm.cm.writeBodyRLPCache(hash, rlp)
```

Comment:
Actually it does not matter that much, however, I think cache should be written after written to the database.

Reply:
Can you elaborate why?
I thought cache should be the one with the recent values, so that it could reduce time for accessing recent values.

Reply:
IMO, cache should be the subset of whole database. It's a subset which provides quick access. If the data is stored to the cache before the the database, we can't guarantee that the data can be retrieved from the database next time. This could cause an inconsistent state, however, in most of our cases, `Crit` will be called inside the database, so it would make little difference. So.. like I said in the above comment, it does not make a big difference and it's a matter of taste.